lib/rspec/expectations/failure_aggregator.rb

```diff
@@ -48,6 +48,8 @@ def call(failure, options)
         @seen_source_ids[source_id] = true
         assign_backtrace(failure) unless failure.backtrace
         failures << failure
+
+        nil
```

Comment:
```suggestion
        FailedMethod
```

Reply:
Where this is declared somewere as:
```
FailedMethod = :a_failed_method_return_value.freeze
```
Or something

Reply:
It could also be a more sophisticated value object (but should still be frozen object).

Reply:
Side note: symbols are frozen, no need to `.freeze` explicitly.

Reply:
Thanks for the review!
I've added an `AggregatedFailure` value object, please take a look. I'll be OK to fall back to a simple symbol if you think it's a better solution